src/main/java/uk/gov/hmcts/probate/services/submit/health/SubmitHealthIndicator.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.hmcts.probate.services.submit.health;
+
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.boot.actuate.health.HealthIndicator;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.HttpStatusCodeException;
+import org.springframework.web.client.ResourceAccessException;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.client.UnknownHttpStatusCodeException;
+
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
```

Comment:
If we're going to introduce lombok maybe we should do it consistently across the whole project